zipkin-ui/templates/trace.mustache

```diff
@@ -137,7 +137,7 @@
         </table>
 
         <button class="btn btn-info btn-xs pull-right" type="button" data-toggle="collapse"
-                data-target="#moreInfo" aria-expanded="false" aria-controls="moreInfo">More Info</button>
+                data-target="#moreInfo" aria-expanded="false" aria-controls="moreInfo" data-i18n="trace.more">More Info</button>
```

Comment:
so is it normal to have the english repeated here and in the resource file? Just asking as I don't know

Reply:
I believe this is so if you opened this file, it could (mostly?) render as normal HTML and it would have values in it. Those values will be replaced when the whole application is running.